src/test/resources/cucumber/features/integrationTests/cli/astCliScan.feature

```diff
@@ -0,0 +1,39 @@
+@AST_CLI_SCAN  @IntegrationTest
+Feature: AST support in CxFlow command-line
+
+    @Skip
+    Scenario Outline: AST CLI scan of a local directory
+        Given scanner is <scanner>
+        And source directory contains vulnerable files
+        When running CxFlow with `scan local sources` options
+        Then bug tracker contains <issueNumber> issues
+        And no exception is thrown
+        Examples:
+            | scanner | issueNumber |
+            #| SCA     | 5           |
```

Comment:
I guess the examples should either be ("AST", "AST,SCA") or just ("AST").
If the SCA line is not needed, please remove it.